buildspecs/runBenchmarks.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
minor: FYI the CodeBuild Docker images all include Python if Bash wasn't the first choice